RELEASE_NOTES.md

```diff
@@ -4,34 +4,304 @@
+  action :uninstall
+end
+```
+
+chocolatey_package correctly setting the :remove action:
+
+```ruby
+chocolatey_package 'nginx' do
+  action :remove
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/LaunchdDeprecatedHashProperty
 
+The `LaunchdDeprecatedHashProperty` cop detects the use of the deprecated `hash` property in the `launchd` resource. The hash property was renamed to `plist_hash` in Chef Infra Client 13 and support for the `hash` name was removed in `Chef Infra Client.
+
+
+`Examples`
+
+launchd with the deprecated `hash` property:
+
+```ruby
+launchd 'foo' do
+  hash foo: 'bar'
+end
+```
+
+launchd with the correct `plist_hash` property:
+```ruby
+launchd 'foo' do
+  plist_hash foo: 'bar'
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
 #### Chef/LocaleDeprecatedLcAllProperty
 
+The `LocaleDeprecatedLcAllProperty` cop detects the use of the `lc_all` property in the `locale` resource. The `lc_all` property was deprecated in Chef Infra Client 15.0 and will be removed in the 16.0 release. Setting the LC_ALL variable is NOT recommended. As a system-wide setting, LANG should provide the desired behavior. LC_ALL is intended to be used for temporarily troubleshooting issues rather than an everyday system setting. Changing LC_ALL can break Chef’s parsing of command output in unexpected ways. Use one of the more specific LC_ properties as needed.
+
+`Examples`
+
+locale resource setting the lc_all property:
+
+```ruby
+locale 'set locale' do
+  lang 'en_gb.utf-8'
+  lc_all 'en_gb.utf-8'
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/UserDeprecatedSupportsProperty
 
+The `UserDeprecatedSupportsProperty` cop detects the usage of the deprecated `supports` property in the `user` resource.
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/PowershellScriptExpandArchive
 
+The `PowershellScriptExpandArchive` op detects the usage of the `powershell_script` resource to run the `Expand-Archive` Cmdlet. The [archive_file](https://docs.chef.io/resource_archive_file.html) resource in Chef Infra Client 15.0 should be used instead.
+
+`Examples`
+
+powershell_script using Expand-Archive to setup a website:
+
+```ruby
+powershell_script 'Expand website' do
+  code 'Expand-Archive "C:\\file.zip" -DestinationPath "C:\\inetpub\\wwwroot\\mysite"'
+  not_if { File.exist?("C:\\inetpub\\wwwroot\\mysite") }
+end
+```
+
+The same archive handled by archive_file:
+
+```ruby
+archive_file 'Expand website' do
+  path 'C:\file.zip'
+  destination 'C:\inetpub\wwwroot\mysite'
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/PowershellInstallPackage
 
+The `PowershellInstallPackage` cop detects the usage of the `powershell_script` resource to run the `Install-Package` Cmdlet. The [powershell_package](https://docs.chef.io/resource_powershell_package.html) resources should be used to install packages via the PowerShell Package Manager instead.
+
```

Comment:
Does this need a title for the code snippet below?

Reply:
Fixed